driver/web/rest/adminapis.go

```diff
@@ -56,6 +56,25 @@ func (h AdminApisHandler) GetCovid19Config(current model.User, group string, w h
 	w.Write(data)
 }
 
+func (h AdminApisHandler) GetCovid19Configs(current model.User, group string, w http.ResponseWriter, r *http.Request) {
+	configs, err := h.app.GetCovid19Configs()
```

Comment:
If you look in the other APIs you will notice that we use the Administration interface. You have to define the GetCovid19Configs() method in the Administration Interface https://github.com/rokwire/health-building-block/blob/3c538d37004057ee1b77bedeb4c66433b6ad0dd5/core/interfaces.go#L294
After that you work with the defined method like this
configs, err := h.app.Administration.GetCovid19Configs()